docs/team/mep.md

```diff
@@ -0,0 +1,265 @@
+MMTk Enhancement Proposal (MEP)
+===============================
+
+An MMTk Enhancement Proposal (MEP) is a more formal variant of issue.  It has a special format, and
+will undergo a more thorough review process.  Its goal is helping the MMTk developers making more
+informed decisions.
+
+MEP is inspired by the Java Enhancement Proposal, described in https://openjdk.org/jeps/1  However,
+unlike JEP which is more open-ended, MEP is more focused on making design decisions.
+
+# When is MEP required?
+
+An MEP is required when making a significant change to the design of the MMTk core.  It is
+applicable to any kind of significant changes, including but not limited to:
+
+-   Bug fixes, including performance bug fixes, that require change to a major part of the MMTk
+    core.
+-   Changes to the MMTk core to implement a feature demanded by bindings.
+-   Major refactoring to the MMTk core.
+
+One purpose of MEP is reducing the risks to the future development of MMTk.  Large-scale changes and
+public API changes usually indicate such risks, but these are only indicators, not criteria.  The
+assessment of risks is subjective, and we need to discuss in order to reach consensus.
+
+Note: JEP is also required for things that "require two or more weeks of engineering effort" and/or
```

Comment:
I would suggest not mentioning too much about JEP, especially when it is irrelevant. 1. As you said, MEP is clearly different from JEP. 2. Not many people are that familiar with JEP so it is hard for them to draw connection between the two.

Reply:
I agree.  I removed all places that mention JEP, keeping only one sentence that mentions that MEP is inspired by JEP.